client/my-sites/plugins/with-plugin-redirect/index.js

```diff
@@ -157,11 +157,14 @@ const withPluginRedirect = createHigherOrderComponent(
 					return ( window.location.href = shouldRedirect );
 				}, 6000 );
 
-				const redirectMessage = translate( 'Redirecting to setup %(pluginName)s in five seconds.', {
-					args: {
-						pluginName: redirectHandler.name,
-					},
-				} );
+				const redirectMessage = translate(
+					'Redirecting to the setup for %(pluginName)s in five seconds.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Redirecting to setup for %(pluginName)s in five seconds.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/with-plugin-redirect-i18n)